src/renderers/webgl/WebGLBackground.js

```diff
@@ -18,6 +18,7 @@ function WebGLBackground( renderer, state, objects, premultipliedAlpha ) {
 
 	var planeMesh;
 	var boxMesh;
+	var previousBackground;
```

Comment:
We would probably call that `currentBackground`. It only needs to be reset when it changes -- not every frame.

Reply:
:+1: